clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -673,6 +673,8 @@ public void close() {
                 log.error("Error when closing " + reporter.getClass().getName(), e);
             }
         }
+
+        log.info("Metrics closed");
```

Comment:
Would it be reasonable to add a message before closing each individual reporter?